extensions-contrib/opencensus-extensions/src/main/java/org/apache/druid/data/input/opencensus/protobuf/OpenCensusProtobufInputRowParser.java

```diff
@@ -88,17 +110,28 @@ public OpenCensusProtobufInputRowParser withParseSpec(ParseSpec parseSpec)
       throw new ParseException(e, "Protobuf message could not be parsed");
     }
 
+    // Fetch metric name.
+    String metricName = this.hasMetricDimension ? this.metricDimension : metric.getMetricDescriptor().getName();
+
+    // Process metric descriptor labels map keys.
+    List<String> descriptorLabels = metric.getMetricDescriptor().getLabelKeysList().stream()
+        .map(s -> (!this.hasMetricLabelPrefix ? s.getKey() : this.metricLabelPrefix + s.getKey()))
```

Comment:
as mentioned above if we set the prefix to empty by default if not set, we can avoid the complexity here.

Reply:
Done.